api/database/Profiledb.js

```diff
@@ -1,23 +1,23 @@
 import sqlite3 from "sqlite3";
-export let db = new sqlite3.Database("./Profile.db", (err) => {
+export let profile_db = new sqlite3.Database("./Profile.db", (err) => {
   if (err) {
-    console.log("Database Connection error: ", err.message);
+    console.log("Profile Database Connection error: ", err.message);
     throw err;
   } else {
-    console.log("Connected to the SQLite db.");
-    db.run(
+    console.log("Connected to the Profile SQLite db.");
+    profile_db.run(
       `CREATE TABLE IF NOT EXISTS Profile (
       id INTEGER PRIMARY KEY AUTOINCREMENT,
       bio TEXT,
       age INTEGER
     )`,
       (err) => {
         if (!err) {
-          console.log("Table is ready. Inserting initial data.");
+          console.log("Profile Table is ready. Inserting initial data.");
 
           let insert = "INSERT INTO Profile (bio, age) VALUES (?,?)";
-          db.run(insert, ["These are my fat nuts", 30]);
-          db.run(insert, ["I cant wait to eat ass", 22]);
+          profile_db.run(insert, ["These are my fat nuts", 30]);
+          profile_db.run(insert, ["I cant wait to eat ass", 22]);
```

Comment:
Why are these in snake case instead of camel case?

Reply:
Because i didnt like the way they looked when camel, it wasnt easy for me to read

-profileDB 
-profileDb
-dbProfile
-DBProfile
-dbprofile
snake case was just easier on the eyes imo

Reply:
@GrpFruit does this resolve your comment?

Reply:
Makes sense it just throughs naming conventions out the window